lib/facter/lvm_support.rb

```diff
@@ -4,8 +4,8 @@
   confine :kernel => :linux
 
   setcode do
-    vgdisplay = Facter::Util::Resolution.which('vgs')
```

Comment:
The other test is failing because this function is no longer being called. I think you need to remove line 24 of `spec/unit/facter/lvm_support_spec.rb`.